helm-chart/renku-notebooks/values.yaml

```
@@ -156,6 +156,9 @@ sessionAutosave:
   ## How long should k8s wait for an autosave branch to be created before
   ## fully and forcefully removing a user session
   terminationGracePeriodSeconds: 600
+  ## How long to wait before showing a warning to the user that their session
+  ## is about to be terminated
```

```suggestion
  ## The warning for session termination shows up this many seconds before the
  ## session is terminated due to idleness.
```

The way that you have it makes me think that if we will kill the session in 12 hours we start showing the warning 11 hours before the session is terminated. But we would only show the warning 1 hour before the session is terminated.

The default value here should be larger. 1 hours is not enough. I would set it to at least 8 or 12 hours. I.e. so that if someone cares they can catch the fact that their session will be killed as they are leaving school/work for the day.